.gitpod.yml

```
@@ -10,9 +10,12 @@ tasks:
   - name: Run backend
     command: |
       gp sync-await setup
-      mkdir -p custom/conf/
-      echo -e "[server]\nROOT_URL=$(gp url 3000)/" > custom/conf/app.ini
-      echo -e "\n[database]\nDB_TYPE = sqlite3\nPATH = $GITPOD_REPO_ROOT/data/gitea.db" >> custom/conf/app.ini
+      if [[ ! -f custom/conf/app.ini ]]
```

```suggestion
      if [ ! -f custom/conf/app.ini ]
```
I don't see a need to use the `bashism` if the POSIX compliant version works too…

---

BTW bash is guaranteed in Gitpod